mmv1/third_party/terraform/tests/resource_compute_instance_group_manager_test.go.erb

```diff
@@ -1490,10 +1518,17 @@ resource "google_compute_instance_group_manager" "igm-basic" {
     delete_rule = "NEVER"
   }
 
-  stateful_disk {
-    device_name = "my-stateful-disk2"
+<% unless version == "ga" -%>
+  stateful_internal_ip {
+    interface_name = "nic0"
```

Comment:
It looks like `interface_name` is updatable on both internal and external ips - but changing that would probably be seen as a "deletion" by the API because its data structure is different. So, just noting this for myself / future folks - nothing to change here.